autoupdate.services/test/unit/src/org/netbeans/api/autoupdate/UpdateProviderFactoryCreateTest.java

```diff
@@ -50,7 +49,7 @@ protected void tearDown () throws  Exception {
     public void testCreate () throws Exception {
         String name = "new-one";
         String displayName = "Newone";
-        URL url = UpdateUnitFactoryTest.class.getResource ("data/catalog.xml");
+        URL url = getClass().getResource ("/org/netbeans/modules/autoupdate/updateprovider/data/catalog.xml");
```

Comment:
Doesn't the relative path data/catalog.xml work anymore?

Reply:
There's no catalog.xml here : https://github.com/apache/incubator-netbeans/tree/master/autoupdate.services/test/unit/src/org/netbeans/api/autoupdate

Reply:
no data/catalog.xml in above location

Reply:
Using the one from https://github.com/apache/incubator-netbeans/tree/master/autoupdate.services/test/unit/src/org/netbeans/modules/autoupdate/updateprovider/data

Reply:
Can you please approve this?

Reply:
I didn't know you were waiting for me. Sure, it looks good!

Reply:
OK, will merge this one.